applications/ParticleMechanicsApplication/python_scripts/mpm_solver.py

```diff
@@ -0,0 +1,474 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications and dependencies
+import KratosMultiphysics.ParticleMechanicsApplication as KratosParticle
+
+# Importing the base class
+from KratosMultiphysics.python_solver import PythonSolver
+
+def CreateSolver(model, custom_settings):
+    return MPMSolver(model, custom_settings)
+
+class MPMSolver(PythonSolver):
```

Comment:
Can or should we go for the structural mechanics here? Also @philbucher @RiccardoRossi

Reply:
we can discuss this tomorrow, but I personally don't want to do that.